HunterPie.Integrations/Datasources/MonsterHunterRise/Entity/Player/MHRWirebug.cs

```diff
@@ -67,6 +68,19 @@ private set
         }
     }
 
+    public string WirebugCondition
+    {
+        get => _wirebugCondition;
+        private set
+        {
+            if (value != _wirebugCondition)
+            {
+                _wirebugCondition = value;
+                this.Dispatch(_onWirebugConditionChange, this);
+            }
+        }
+    }
```

Comment:
Conditions, States, anything that has multiple values within a possible set of options should be an enum, not  a string.

Reply:
It using WirebugCondition enum.

Reply:
Renamed WirebugConditon to WirebugState